activerecord/test/cases/adapters/sqlite3/sqlite3_adapter_test.rb

```diff
@@ -284,9 +284,9 @@ def test_transaction
 
```

Comment:
Same as `activerecord/test/cases/adapter_test.rb` and `activerecord/test/cases/adapters/postgresql/connection_test.rb`, we can not change.